spec/tutorials/discogs_api_tutorial/models/artist_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Artist, type: :model do
+  let(:user) { create(:user) }
+  let(:artist) { create(:artist, user: user) }
+  let!(:release) { create(:release, artist: artist)}
```

Comment:
This `bang` is really necessary?

Reply:
The `release` variable is called only after the expectation has been built. I could remove the `!` and put a `before` block. Or I could create a factory called `artist_with_releases`. But I believe these things are too much. WDYT?

Reply:
Since we don't need to work with this variable directly, I would move to a `#before` block to be more explicit.

Reply:
Since we're working with this variable only once, I would move to its expectation block.